app/controllers/staff/training_sessions_controller.rb

```diff
@@ -2,6 +2,7 @@ class Staff::TrainingSessionsController < StaffAreaController
 
   before_action :current_training_session, only: [:show, :edit, :update, :certify_trainees, :destroy]
   before_action :changed_params, only: [:update]
+  before_action :verify_ownership, only: [:show]
```

Comment:
Shouldn't this be on everything except show? Like shouldn't this apply for delete and update too?